### ISSUE TITLE
License is required if ephemeral context is enabled

### ISSUE THREAD

**Issue by ViliusS** [AxonIQ/axon-server-se#698]
According to [Axon Server pricing](https://www.axoniq.io/pricing/axon-server) ephemeral context is available in developer/starter editions, however when we try to use it we get the following error:

```
2024-07-09 07:22:20.456 ERROR 3885617 — [grpc-executor-4] i.a.axonserver.grpc.LicenseInterceptor : Warning: Unauthorized feature(s) that cannot be used with this license have been detected. Please remove them or contact AxonIQ for more information/assistance.
```

and on the client:
```
2024-07-09 07:22:20.549 ERROR [main] org.springframework.boot.SpringApplication : Application run failed

io.grpc.StatusRuntimeException: CANCELLED: License is not valid!
```

When a server is restarted we also get:
```
2024-07-09 08:44:56.046 ERROR 4055503 --- [           main] i.a.axonserver.LicenseAccessController   : Feature violation detected:
2024-07-09 08:44:56.047 ERROR 4055503 --- [           main] i.a.axonserver.LicenseAccessController   :  Number of nodes should be reduced: false, Number of contexts should be reduced: false, Number of applications should be reduced: false, Multi-Tier context should be removed: true
```

This is on latest Axon Server 2024.0.4 docker image with just `AXONIQ_AXONSERVER_STANDALONE` variable set to `true`. After server is started we just delete old `default` context and create new `default` with ephemeral option enabled.

Is it a bug or are we doing something wrong?

**Comment by schananas** [comment]
`After server is started we just delete old default context and create new default with ephemeral option enabled.`

Even if you have one 'default' ephemeral context you still get the error, is they correct?

**Comment by ViliusS** [comment]
Yes, as you can see from the last error log we only have one context:

2024-07-09 08:44:56.047 ERROR 4055503 --- [           main] i.a.axonserver.LicenseAccessController   :  Number of nodes should be reduced: false, **Number of contexts should be reduced: false**, Number of applications should be reduced: false, **Multi-Tier context should be removed: true**

**Comment by schananas** [comment]
It seams it's a bug, we have already created a fix but it will take a week or few to release a patch.
We can issue a temp licence for you to use and mitigate this issue, it can be standalone or console included. Would this work for you?

**Comment by ViliusS** [comment]
Thank you for confirming. If that's few weeks, we can wait.

**Comment by ViliusS** [comment]
Can confirm this as fixed in 2024.1.0.
